detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/ProjectInspector.cs

```diff
@@ -191,6 +191,10 @@ public Container GetContainer()
                         Console.WriteLine("Reference resolver succeeded.");
                         projectNode.Packages = projectReferencesResult.Packages;
                         projectNode.Dependencies = projectReferencesResult.Dependencies;
+                        if (projectNode.Dependencies.Count == 0 && projectNode.Packages.Count == 0 && projectReferencesResult.BadParse)
```

Comment:
Is it not enough to fail if BadParse is true?

Reply:
Yup, should be enough, sorry forgot to change it.